components/raftstore/src/coprocessor/dispatcher.rs

```diff
@@ -9,12 +9,17 @@ use engine_traits::{CfName, KvEngine};
 use kvproto::metapb::Region;
 use kvproto::pdpb::CheckPolicy;
 use kvproto::raft_cmdpb::{ComputeHashRequest, RaftCmdRequest};
+use lazy_static::lazy_static;
 use raft::eraftpb;
 use tikv_util::box_try;
 
 use super::*;
 use crate::store::CasualRouter;
 
+lazy_static! {
+    static ref EMPTY_REGION: Region = Region::default();
```

Comment:
Why not use `Region::default_instance()`?

Reply:
Done.